lib/ruby_smb/dcerpc/netlogon/netr_server_authenticate3_response.rb

```diff
@@ -0,0 +1,26 @@
+require 'ruby_smb/dcerpc/ndr'
+
+module RubySMB
+  module Dcerpc
+    module Netlogon
+
+      # [3.5.4.4.2 NetrServerAuthenticate3 (Opnum 26)](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-nrpc/3a9ed16f-8014-45ae-80af-c0ecb06e2db9)
+      class NetrServerAuthenticate3Response < BinData::Record
+        attr_reader :opnum
+
+        endian :little
+
+        netlogon_credential :server_credential
+        uint32              :negotiate_flags
+        uint32              :account_rid
+        uint32              :status
```

Comment:
Not a blocker, but all the other RPC structures in `ruby_smb` use `error_status` instead of `status`. I think it would be interesting to follow the same name convention.